UI/window-basic-main.cpp

```diff
@@ -8902,9 +8903,10 @@ void OBSBasic::SetShowing(bool showing)
 
 		if (showHide)
 			showHide->setText(QTStr("Basic.SystemTray.Show"));
-		QTimer::singleShot(250, this, SLOT(hide()));
+		QTimer::singleShot(0, this, SLOT(hide()));
```

Comment:
Does this actually still need to be in a QTimer if it's being triggered at a delay of 0, or is this more "rather not break anything unnecessarily" territory?

Reply:
Yes, as sometimes the window would hang when showing/hiding, so it needed to be toggled on the next event loop.